source/background/library/storageMigration.js

```diff
@@ -0,0 +1,31 @@
+import { TASK_TYPE_HIGH_PRIORITY } from "@buttercup/channel-queue";
+import log from "../../shared/library/log.js";
+import { getDefaultStorageAdapter } from "./BrowserStorageInterface.js";
+
+export function migrateLocalStorageToChromeStorage(queue) {
+    log.info("(Migration)", "Beginning data migration for new storage");
```

Comment:
Can we add a check here if it's already migrated dont run this or is it not needed because you delete the keys?

Reply:
Good catch! But yes, already handled due to the keys being removed on [line 27](https://github.com/buttercup/buttercup-browser-extension/pull/114/files/31f37bacbe924abaf842eb03754c561e3fda8e4d#diff-cd1cbc335cc70d4bfb219db1a00adec5R27).